spec/support/features/in_person_helper.rb

```diff
@@ -74,6 +91,14 @@ def complete_state_id_step(_user = nil)
     click_idv_continue
   end
 
+  def complete_state_id_step_when_id_and_residential_address_differ(_user = nil)
```

Comment:
Similar to previous comment I'd consider using a couple of arguments here instead of a new function, like

```
def complete_state_id_step(_user = nil, include_address: false, same_address_as_id: true)
```

And then to use it do `complete_state_id_step(user, include_address: true, same_address_as_id: false)`